docs/articles/documentation/using-testcafe/programming-interface/runner.md

```diff
@@ -64,16 +64,20 @@ src(source) → this
 
 Parameter | Type                | Description
 --------- | ------------------- | ----------------------------------------------------------------------------
-`source`  | String &#124; Array | The relative or absolute path to a test fixture file, or several such paths.
+`source`  | String &#124; Array | The relative or absolute path to a test fixture file, or several such paths. You can use `glob` patterns to include (or exclude) multiple files.
```

Comment:
Can we add some link to `glob` description?

Reply:
Yes, we can. We already do this [here](http://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#file-pathglob-pattern). 
@VasilyStrelyaev the feature was implemented. You can continue to work on this